### PR TITLE
PAL-1077 updated image repository property

### PR DESCRIPTION
**Pull request by pd104923**
- Updated image repository value in parent pom

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
         <encoding>UTF-8</encoding>
         <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
-        <image.repository/>
+        <image.repository>gchq/palisade-</image.repository>
```
Comment (m78233) [inline pom.xml:49]: Question, can we use this without the "-" at the end of the name?

**Review by m78233 — COMMENTED** [review]
One question.

File: pom.xml (hunk 1 above)
Comment (pd104923) [reply pom.xml:49]: The "-" is the separator as the image will be `gchq/palisade-{service image}`

**Review by developer01189998819991197253 — APPROVED** [review]

**Comment by ac74475** [comment]
Readers does not build docker images, therefore the image.repository maven value is not required. Can you remove the property rather than updating the value.

**Review by m78233 — APPROVED** [review]

**Comment by pd104923** [comment]
> Readers does not build docker images, therefore the image.repository maven value is not required. Can you remove the property rather than updating the value.

The property has been removed from the parent pom

**Review by ac74475 — APPROVED** [review]

**Pull request merged**